pytensor/graph/basic.py

```diff
@@ -907,6 +907,54 @@ def graph_inputs(
     yield from (r for r in ancestors(graphs, blockers) if r.owner is None)
 
 
+def explicit_graph_inputs(
+    graph: Variable[Any, Any] | Iterable[Variable[Any, Any]],
+) -> Generator[Variable, None, None]:
+    """
+    Get the inputs into PyTensor variables
+
+    Parameters
+    ----------
+    graph: TensorVariable
```

Comment:
```suggestion
    graph : TensorVariable
```

there must be a space before and after the colon to render correctly